src/Business Foundation/App/NoSeries/src/Setup/NoSeriesSetupImpl.Codeunit.al

```diff
@@ -15,6 +15,7 @@ codeunit 305 "No. Series - Setup Impl."
         NumberFormatErr: Label 'The number format in %1 must be the same as the number format in %2.', Comment = '%1=No. Series Code,%2=No. Series Code';
         UnIncrementableStringErr: Label 'The value in the %1 field must have a number so that we can assign the next number in the series.', Comment = '%1 = New Field Name';
         NumberLengthErr: Label 'The number %1 cannot be extended to more than 20 characters.', Comment = '%1=No.';
+        CodeFieldChangedErr: Label 'The filter on %1 was altered by an event subscriber. This is a programming error. Please contact your partner to resolve the issue.\Original %1: %2\Modified Filter: %3';
```

Comment:
As the error below shows, whenever you add a label with placeholder, you must add a comment specifying what those placeholders are. These are necessary for translation teams to translate the string better. This is required for all strings with placeholders because some string can be very difficult for a translator to understand when they almost fully consist of these placeholders..